examples/python/structure_from_motion/main.py

```diff
@@ -110,7 +110,7 @@ def read_and_log_sparse_reconstruction(dataset_path: Path, filter_output: bool,
         points3D = {id: point for id, point in points3D.items() if point.rgb.any() and len(point.image_ids) > 4}
 
     rr.log("description", rr.TextDocument(DESCRIPTION, media_type=rr.MediaType.MARKDOWN), timeless=True)
-    rr.log_view_coordinates("/", up="-Y", timeless=True)
+    rr.log("/", rr.ViewCoordinates.RIGHT_HAND_Y_DOWN, timeless=True)
```

Comment:
Somehow, for some reason I don't understand, in this specific case logging `ViewCoordinates` at `/` works, despite #3538